packages/react/src/ResourceBlame/ResourceBlame.test.tsx

```diff
@@ -43,17 +47,20 @@ describe('ResourceBlame', () => {
   });
 
   test('getTimeString', () => {
-    expect(getTimeString(new Date(Date.now() - 1e3).toUTCString())).toEqual('1 second ago');
-    expect(getTimeString(new Date(Date.now() - 2e3).toUTCString())).toEqual('2 seconds ago');
-    expect(getTimeString(new Date(Date.now() - 60e3).toUTCString())).toEqual('1 minute ago');
-    expect(getTimeString(new Date(Date.now() - 120e3).toUTCString())).toEqual('2 minutes ago');
-    expect(getTimeString(new Date(Date.now() - 3600e3).toUTCString())).toEqual('1 hour ago');
-    expect(getTimeString(new Date(Date.now() - 7200e3).toUTCString())).toEqual('2 hours ago');
-    expect(getTimeString(new Date(Date.now() - 86400e3).toUTCString())).toEqual('1 day ago');
-    expect(getTimeString(new Date(Date.now() - 172800e3).toUTCString())).toEqual('2 days ago');
-    expect(getTimeString(new Date(Date.now() - 2592000e3).toUTCString())).toEqual('1 month ago');
-    expect(getTimeString(new Date(Date.now() - 5184000e3).toUTCString())).toEqual('2 months ago');
-    expect(getTimeString(new Date(Date.now() - 31536000e3).toUTCString())).toEqual('1 year ago');
-    expect(getTimeString(new Date(Date.now() - 63072000e3).toUTCString())).toEqual('2 years ago');
+    const now = Date.now();
+    const dateSpy = jest.spyOn(global.Date, 'now').mockImplementation(() => now);
```

Comment:
You might be able to use [Jest fake timers](https://jestjs.io/docs/jest-object#fake-timers) to do this a bit more ergonomically

Reply:
Thanks for the tip! I'll take a quick look and see if I can get that working.